### PR TITLE
Quick PEP8-ish reformating pass

### PR DESCRIPTION
**Pull request by kellybyrd**
Let SublimeLinter and isort guide me on some reformatting, but
intentionally ignored things like 2 vs 4 spaces for indents and
too many statements on a line.

Also added `__main__` check, but it's not really needed right now

### CODE REVIEW

File: nearby.py [hunk 1]
```diff
@@ -5,67 +5,70 @@
...
-if len(data):
-  print(f"\nhotspot name                   count    rssi  witness time")
-  for h in sorted(data, key=lambda x: x.get('recent_time')):
-    # h = dict of pocs that saw us
-    hist = h['hist']
-    rssi = max(hist, key=hist.get)
-    count = sum(hist.values())
-    print(f"{h['name']:25} {count:10}   {rssi:5}  {datetime.datetime.fromtimestamp(h['recent_time']/1000**3).isoformat(timespec='minutes')}")
 
+if __name__ == '__main__':
```
Comment (tedder) [inline nearby.py]: to save me the indentation work, can you just change this to a monolithic function and move the `__main__` to the bottom? It doesn't have to be perfectly architected, just a basic call.

(if that doesn't make sense I will do it after the merge)

Comment (kellybyrd) [reply nearby.py]: Heh. I almost did that, but have run across folks who didn't like a `def main:` Doing that now.

Comment (kellybyrd) [reply nearby.py]: I think that's what you were asking for. If not, feel free to modify however you want

**Comment by tedder** [comment]
perfect, good start on the cleanup. thx :)

**Pull request merged**